### DISCUSSION TITLE
Cant get it to run on linux

### DISCUSSION THREAD

**Discussion by BlackHeart-TF** [unoplatform/uno#12482] [Q&A]
So far i have been completely unsucesful at getting Uno to run on linux and currently have no idea how to proceed.
IM just working with a clean "dotnet new unoapp" the Windows version seems to run fine. i also got it working on WSL with ubuntu in it and vcxsrv. the same commands within WSL did not give the same results on a regular ubuntu 22 machine.

the most success ve had so far has been building with `dotnet build unoapp.Skia.Gtk/unoapp.Skia.Gtk.csproj -v normal -c Release  --self-contained --runtime debian.8-x64 -o out`
and running with `dotnet unoapp.Skia.Gtk.dll`
this runs as-is on WSL but on bare metal gives: 
```
Failed to load /server/Tools/_Code/Listi2/out/libhostpolicy.so, error: /server/Tools/_Code/Listi2/out/libhostpolicy.so: failed to map segment from shared object
An error occurred while loading required library libhostpolicy.so from [/server/Tools/_Code/Listi2/out/]
```
not sure what other details would help

**Comment by jeromelaban** [comment]
Thanks for the report. 

This looks like a similar error as reported in https://github.com/dotnet/sdk/issues/8189. It is likely that since you're creating a self contained app, you need to run it using the executable of the app's name (`unoapp.Skia.Gtk` in your example above), not the `.dll`.

**Reply by BlackHeart-TF** [reply]
that just says permission denied, even as root, and verifying it has +x
```
ls -l unoapp.Skia.Gtk
-rwxr-xr-x 1 user user 151064 May 31 10:33 unoapp.Skia.Gtk
```
```
./unoapp.Skia.Gtk
zsh: permission denied: ./unoapp.Skia.Gtk

```
and running the dll or exec both work fine under WSL. interesting. i dont know how to tell what im being denied permission to

**Comment by BlackHeart-TF** [comment]
Ended up being a weird file permission thing from a network share
